app/src/main/res/layout/fragment_game.xml

```diff
@@ -1,6 +1,127 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/purple_500">
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/player_turn_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="18dp"
+            android:textAlignment="center"
+            android:textColor="#8C9EFF"
+            android:textSize="36sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="O's Turn" />
+
+        <ImageView
+            android:id="@+id/image_background"
+            android:layout_width="410dp"
+            android:layout_height="410dp"
+            android:adjustViewBounds="true"
+            android:contentDescription="@string/board"
+            android:src="@drawable/icon_background"
+            app:layout_constraintBottom_toBottomOf="@id/tic_tac_toe_board"
+            app:layout_constraintEnd_toEndOf="@id/tic_tac_toe_board"
+            app:layout_constraintHorizontal_bias="0.45"
+            app:layout_constraintStart_toStartOf="@id/tic_tac_toe_board"
+            app:layout_constraintTop_toTopOf="@id/tic_tac_toe_board"
+            app:layout_constraintVertical_bias="0.48000002" />
```

Comment:
is this supposed to be 0.48 instead of 0.50?

Reply:
I had some difficulty aligning the background image I made with my grid (I attempted a couple of different layouts before settling on GridLayout). So I used the Design tab's attributes bar to align them graphically. 

That would likely explain that value. I will see how it looks if I update it.

Initially, I made a working layout on my test phone, but later realized it didn't size correctly on other devices. So even though GridLayout is older, and seemed to be discouraged, I figured in this use case it was ok because it always remains on screen and it doesn't load any images until a square is clicked by a user.

If I make changes in my local branch and push them back up to remote, would I be able to edit this PR with them?

Reply:
Actually, can I edit them from within the PR remotely and pull the changes back down?

Reply:
both definitely work, but generally you'd edit the values locally, and push up a new commit